### PR TITLE
Update viewing monitor results for v8

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/monitoring-your-api/faqs-monitors.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ There is no limit to the number of monitors you can create. You can have any num
 
 ### What timezone is my monitor set to?
 
-Your monitor's timezone is automatically set to the timezone of your computer at the time of the monitor's creation, unless you [define the regions](/docs/postman/monitors/setting-up-monitor/#adding-regions). You can find the auto-selected region by [viewing your monitor in the Dashboard](/docs/postman/monitors/viewing-monitor-results/#viewing-individual-monitors).
+Your monitor's timezone is automatically set to the timezone of your computer at the time of the monitor's creation, unless you [define the regions](/docs/postman/monitors/setting-up-monitor/#adding-regions). You can find the auto-selected region by [viewing your monitor in the Dashboard](/docs/postman/monitors/viewing-monitor-results/#viewing-monitors-in-postman).
```
Comment (ArjunSingh-PM) [inline src/pages/docs/designing-and-developing-your-api/monitoring-your-api/faqs-monitors.md:25]: The schedule that the monitor runs at is set to the timezone of the computer the monitor was created from. I doubt regions have a role to play in this. 

@godfrzero Can you please confirm this?

File: src/pages/docs/designing-and-developing-your-api/monitoring-your-api/viewing-monitor-results.md [hunk 2]
```diff
@@ -163,7 +153,7 @@ You can use this console to both troubleshoot issues and learn more about an ind
 
 You can view a monitor's activity logs by selecting **...** in the upper-right corner > **View activity logs**.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/designing-and-developing-your-api/monitoring-your-api/viewing-monitor-results.md:154]: This is not true.

A user can access the monitor "Activity feed" by clicking on the 'clock' icon in the context bar.

Comment (ArjunSingh-PM) [reply src/pages/docs/designing-and-developing-your-api/monitoring-your-api/viewing-monitor-results.md:154]: We should also add that the user can view monitor details by clicking on the 'info' icon in the context bar. Also, we can explain its content.

File: src/pages/docs/designing-and-developing-your-api/monitoring-your-api/viewing-monitor-results.md [hunk 3]
```diff
@@ -135,25 +125,25 @@ Click to open the drop-down menu **Average**, then select an option. To view the
...
 
 You can view **Test Results** below the monitor summary to find more detailed information on your tests, including which passed or failed, response codes, and response times.
 
-[![test results](https://assets.postman.com/postman-docs/test-results-2.jpg)](https://assets.postman.com/postman-docs/test-results-2.jpg)
+[![test results](https://assets.postman.com/postman-docs/monitor-view-test-results0.jpg)](https://assets.postman.com/postman-docs/monitor-view-test-results0.jpg)
 
-> If your monitor is configured to run in multiple regions, you can view the test results for a particular region by selecting that region from the dropdown to the right of the **Test Results** tab.
+> If your monitor is configured to run in multiple regions, you can view the test results for a particular region by selecting that region from the dropdown to the right of the **Test Results**.
 
 ### Console log
```
Comment (ArjunSingh-PM) [inline src/pages/docs/designing-and-developing-your-api/monitoring-your-api/viewing-monitor-results.md:140]: We can also highlight that the requests in the console log are linked to quickly view the request in a tab and make edits if needed.

**Review by ArjunSingh-PM — CHANGES_REQUESTED** [review]
The path to reach monitoring 'Activity feed' needs edits. Looks good otherwise!

File: src/pages/docs/designing-and-developing-your-api/monitoring-your-api/faqs-monitors.md (hunk 1 above)
Comment (godfrzero) [reply src/pages/docs/designing-and-developing-your-api/monitoring-your-api/faqs-monitors.md:25]: Yep, the schedule is set based on the timezone from which it was created. Regions don't matter.

File: src/pages/docs/designing-and-developing-your-api/monitoring-your-api/viewing-monitor-results.md [hunk 4]
```diff
@@ -135,38 +127,44 @@ Click to open the drop-down menu **Average**, then select an option. To view the
...
+[![console log](https://assets.postman.com/postman-docs/monitor-view-console-log0.jpg)](https://assets.postman.com/postman-docs/monitor-view-console-log0.jpg)
 
 > If your monitor is configured to run in multiple regions, you can view the console logs for a particular region by selecting that region from the dropdown to the right of the **Console Log** tab.
 
 You can use this console to both troubleshoot issues and learn more about an individual run's behavior.
 
 ### Activity log
 
-You can view a monitor's activity logs by selecting **...** in the upper-right corner > **View activity logs**.
+You can view a monitor's activity logs by selecting the clock icon in the upper-right corner > **View activity logs**.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/designing-and-developing-your-api/monitoring-your-api/viewing-monitor-results.md:156]: Not sure if we should call this the 'clock' icon. Can we use the actual icon/picture instead?

**Review by ArjunSingh-PM — APPROVED** [review]
Looks good!

**Review by jerry-li-dev — APPROVED** [review]

**Pull request merged**